<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EclipseTest</class>
 <widget class="QMainWindow" name="EclipseTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EclipseTest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="displayButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="liveViewLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>901</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateLineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
